kuttl-test.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# EXECUTION (from install_yamls repo root):
+#
+#   make openstack_kuttl
+#
+# ASSUMPTIONS:
+#
+# 1. Latest version of kuttl is installed at /usr/local/bin/kubectl-kuttl
+#    - wget https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
+#    - mv kubectl-kuttl_0.11.1_linux_x86_64 /usr/local/bin/kubectl-kuttl
+#    - chmod 755 /usr/local/bin/kubectl-kuttl
+# 2. An OCP 4.10+ CRC cluster with Podified Operators has been deployed
+# 3. CLI user has access to $KUBECONFIG
+# 4. The environment variable INSTALL_YAMLS is set to the the path of the
+#    install_yamls repo
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-openstack
+namespace: openstack
```

Comment:
I think we're trying to use unique namespaces going forward for all KUTTL tests [1].  If we could get that working here, that would be awesome.  Attempting this would also require changing [2] to not `oc apply` against the `openstack` namespace.

[1] https://github.com/openstack-k8s-operators/keystone-operator/blob/769ebc6d0776ca4edf06a2d2e0d9eedcf3906cbb/kuttl-test.yaml#L20
[2] https://github.com/openstack-k8s-operators/openstack-operator/pull/323/files#diff-d760e3bc41b0c740ee3a351bc381a27357a90ef627adaddd61937eaed30020a2

Reply:
@abays yes, that's right, thanks for pointing it out